composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^5.5 || ^7.0",
         "doctrine/phpcr-bundle": "^1.1",
-        "doctrine/phpcr-odm": "^1.1",
+        "doctrine/phpcr-odm": "^1.1 || ^2.0",
```

Comment:
afaik `@dev` is used by others depending on this bundle. not using `@dev` is the right thing here. projects that happen to use master of sonata admin will still get the stable phpcr-odm unless they either specifically ask for the 2 version `@dev` or change minimum-stability.